### DISCUSSION TITLE
Dropdown menu to quickly switch between websites in the Settings area

### DISCUSSION THREAD

**Discussion by gregdotca** [plausible/analytics#1149] [Anything Else]
I think it would be incredibly helpful to be able to quickly and easily switch between websites for the current settings that you're viewing.

For example, say you're on the Goals page for one of your websites, but you want to view the Goals page for a different website. Having a quick access dropdown menu would allow you to switch between websites with a couple clicks (click the dropdown menu, select the other website that you want to view the Goals for).

Currently, in order to view the Goals for a different website, you have to click on "Back to stats", click on the website dropdown menu, select the website, click the website dropdown menu again, click on "Site settings", and then click on "Goals".

Having a website switcher would not only save users a whole bunch of clicks, but it would also allow them to quickly go through the different settings sections website-by-website, making it much easier to remember and compare the settings they have setup in the various sections for each website.

**Comment by metmarkosaric** [comment]
thanks @chetcuti, could be nice to add! 

btw you can also get to the site settings from the /sites landing page. hover over your website name and click on the "Settings" icon on the right hand side of your domain name
